$OpenBSD: patch-abcde,v 1.10 2018/08/31 08:48:12 landry Exp $
Fix tracks offsets list gathering, will be fixed for v2.9.3
Upstream commit 8de69dd9476d40b4d61b25532f0a22acd932d765
Index: abcde
--- abcde.orig
+++ abcde
@@ -2110,7 +2110,7 @@ do_discid ()
 				CDPARANOIAAUDIOTRACKS="$TRACKS"
 
 				LEADOUT="$(echo "$CDPARANOIAOUTPUT" | grep -Eo '^TOTAL[[:space:]]+([[:digit:]]+)' | get_last)"
-				OFFSETS="$(echo "$CDPARANOIAOUTPUT" | sed -n -e's/^ .* \([0-9]\+\) \[.*/\1/p')"
+				OFFSETS="$(echo "$CDPARANOIAOUTPUT" | awk '/^ +.*\[/ {print $4}')"
 				makeids
 				;;
 			*)
